tests/e2e/support/api/graph/utils.ts

```diff
@@ -0,0 +1,24 @@
+import { User } from '../../types'
+import { checkResponseStatus, request } from '../http'
+import join from 'join-path'
+
+interface ApplicationEntity {
+  appRoles: AppRole[]
+  displayName: string
+  id: string
+}
+
+interface AppRole {
+  displayName: string
+  id: string
+}
```

Comment:
I didn't find where we use this interface

Reply:
just above this interface line 6
```
interface ApplicationEntity {
  appRoles: AppRole[]
  displayName: string
  id: string
}
```

Reply:
used in the above interface `L:6`
```ts
interface ApplicationEntity {
  appRoles: AppRole[]
  displayName: string
  id: string
}
```